IDE/IAR-EWARM/embOS/SAMV71_XULT/embOS_wolfcrypt_benchmark_SAMV71_XULT/settings/wolfcrypt_benchmark.Debug.driver.xcl

```diff
@@ -0,0 +1,33 @@
+"--endian=little" 
```

Comment:
It might be worth checking if you really need to include any .xcl files. I believe these are just for debugging and are fairly specific to the user and are easily configured by the tool.